package.json

```diff
@@ -6,32 +6,48 @@
   "engines": {
     "node": ">=18.19.0"
   },
+  "author": "Temitope Ayoku & SAS HOF Team",
   "scripts": {
     "start": "node server.js",
     "start:dev": "hof-build watch --env",
     "test": "NODE_ENV=test yarn run test:lint && yarn run test:unit",
     "test:lint": "eslint . --config ./node_modules/eslint-config-hof/default.js",
     "build": "hof-build",
+    "generate:reports": "node ./services/reports/generate_reports.js",
+    "generate:local:reports": "node -r dotenv/config ./services/reports/generate_reports.js",
+    "test:unit": "nyc _mocha \"test/_unit/**/*.spec.js\"",
+    "test:snyk": "snyk config set api=SNYK_TOKEN && snyk test",
     "postinstall": "hof-build"
   },
   "dependencies": {
     "accessible-autocomplete": "^2.0.4",
-    "axios": "^1.6.5",
-    "bl": "^6.0.10",
+    "aws-sdk": "^2.1333.0",
+    "axios": "^1.6.0",
+    "bl": "^6.0.1",
+    "bottleneck": "^2.19.5",
     "busboy": "^1.6.0",
     "bytes": "^3.1.2",
-    "dotenv": "^16.3.1",
-    "eslint-config-hof": "^1.3.1",
+    "csv-parse": "^5.3.8",
     "hof": "~20.4.0",
-    "jquery": "^3.7.1",
+    "ioredis": "^5.3.1",
+    "jimp": "^0.22.7",
+    "jquery": "^3.6.3",
     "lodash": "^4.17.21",
-    "moment": "^2.30.1",
-    "node-cron": "^3.0.3"
+    "moment": "^2.29.4",
+    "ms-nationalities": "^1.0.1",
+    "node-cron": "^3.0.2",
+    "notifications-node-client": "^7.0.0",
```

Comment:
I'd use version 8.0.0 here it is the most up to date

Reply:
Did you test it?

Reply:
Can not really test it out now but tested the server on unit test and it work fine